pkg/agent/datapath/localBridge.go

```diff
@@ -239,6 +241,20 @@ func (l *LocalBridge) BridgeInit() {
 	l.fromLocalRedirectTable, _ = sw.NewTable(FROM_LOCAL_REDIRECT_TABLE)
 	l.fromLocalArpPassTable, _ = sw.NewTable(FROM_LOCAL_ARP_PASS_TABLE)
 	l.fromLocalArpSendToCtrlTable, _ = sw.NewTable(FROM_LOCAL_ARP_TO_CONTROLLER_TABLE)
+	l.ctStateTable, _ = sw.NewTable(CT_STATE_TABLE)
+	l.ctCommitTable, _ = sw.NewTable(CT_COMMIT_TABLE)
+
+	if err := sw.DeleteTable(INPUT_TABLE); err != nil {
+		log.Fatalf("Failed to delete single local bridge vlanInput table, error: %v", err)
+	}
+
+	if err := sw.DeleteTable(CT_STATE_TABLE); err != nil {
+		log.Fatalf("Failed to delete single local bridge ctState table, error: %v", err)
+	}
+
+	if err := sw.DeleteTable(CT_COMMIT_TABLE); err != nil {
+		log.Fatalf("Failed to delete single local bridge ctCommit table, error: %v", err)
+	}
 
 	if err := l.initVlanInputTable(sw); err != nil {
 		log.Fatalf("Failed to init local bridge vlanInput table, error: %v", err)
```

Comment:
We will lose connectivity if drop the whole default table until L259 ( initialize new vlanInputTable ). As @echkenluo  says, if something fatal, default table 0 may be empty. We would better delete ct flows accurately.

Reply:
has been modified to delete the accurate flows with the specified priority